recipes/pyspark/setup.py

```diff
@@ -24,5 +24,8 @@
       version='0.1.0',
       packages=['pysparkretailapp'],
       install_requires=required,
+      #we will use public Adobe github repo for sdk
+      dependency_links=[
+          'git+ssh://git@git.corp.adobe.com/ml/model-authoring-sdk-pyspark.git@master#egg=model_authoring_sdk'],
```

Comment:
I'm confused here - 'public Adobe github' and internal host (`corp.adobe.com`).

Reply:
Yes, this part is not really required. It was used during testing. It's removed. Thanks.